config/initializers/sentry.rb

```diff
@@ -3,15 +3,4 @@
 
   config.current_environment = ENV['SENTRY_ENV'] || Rails.env
   config.sanitize_fields = ["credentials"]
-
-  config.excluded_exceptions += ['ActionController::RoutingError',
-                                'ActiveRecord::ConcurrentMigrationError',
-                                'Stoplight::Error::RedLight',
```

Comment:
Removing this one will break Stoplight's intended functionality, as this is the exception that's raised when the light is red and the enclosed code is not supposed to be running.

Reply:
won't we just get to know about it in sentry now but the error will still raise internally?

also if this breaks system behaviour should this have been approved?

Reply:
Yeah, this is just filtering sentry payloads, not catching the exceptions. But the purpose of adding Stoplight was to prevent sending a bunch of already-reported errors by raising `Stoplight::Error::RedLight` instead of whatever the original was and filtering on it here.

So as to your second question, it depends on your intentions with this PR. The purpose as I saw it was to send a bunch of errors to Sentry that we're not currently sending. It will certainly do that, I'm just pointing out that it takes Stoplight out of the loop. If you're not worried about that, then this is fine.

Reply:
thanks - do you think there is value in seeing these `Stoplight::Error::RedLight` errors or will they just be noise?

Reply:
They'll always be the result of other errors we're also seeing and will all be combined together in Sentry, but if we've got the bandwidth I guess it can't hurt. Volume is a useful metric, even if we'll have to  figure out how to sort them out by args or stack trace or something else.

Reply:
Let's run without them and we can add them back in if they are noise to us. Thanks for the insightful comments.